web/modules/custom/feedback/feedback.module

```diff
@@ -1,14 +0,0 @@
-<?php
-
-use Drupal\Core\Routing\RouteMatchInterface;
-use Drupal\Core\Form\FormStateInterface;
-
-/**
- * Implements hook_help().
- */
-function feedback_help ($route_name, RouteMatchInterface $route_match) {
```

Comment:
we don't need this file at all. It implements only hook_help which doesn't make sense in its current implementation